### ISSUE TITLE
can yolo detect the words well?

### ISSUE THREAD

**Issue by cdragon2** [thtrieu/darkflow#608]
there are 10 words like dog, cat, elephant, tiger and so on.

per each words, there are 9 images. 

the word put on the same background, depending on 9 areas.

As the result, the loss is about 0.01.

it can detect well if i test by using already trained datasets.

but in the case of new image like putting on similar area, the result not good.

is there some limitation for detecting words?

**Comment by sandeeprepakula** [comment]
I guess you have very less data and the model trained is overfitting to your dataset. Try adding more data to training.
